cdap-security/src/main/java/io/cdap/cdap/security/impersonation/SecurityUtil.java

```diff
@@ -123,14 +123,27 @@ public static String expandPrincipal(@Nullable String principal) throws UnknownH
   }
 
   /**
+   * Check if {@link Constants.Security#KERBEROS_ENABLED} is set. The value is default to the value of
+   * {@link Constants.Security#ENABLED}.
+   *
    * @param cConf CConfiguration object.
    * @return true, if Kerberos is enabled.
    */
   public static boolean isKerberosEnabled(CConfiguration cConf) {
-    return cConf.getBoolean(Constants.Security.KERBEROS_ENABLED,
-                            cConf.getBoolean(Constants.Security.ENABLED) &&
-                              cConf.getEnum(Constants.Security.Authentication.AUTHENTICATION_MODE,
-                                            AuthenticationMode.MANAGED).equals(AuthenticationMode.MANAGED));
+    return cConf.getBoolean(Constants.Security.KERBEROS_ENABLED, cConf.getBoolean(Constants.Security.ENABLED));
```

Comment:
For non-hadoop environments, this will be a no-op right? Can you add a comment clarifying this? Other than this, LGTM

Reply:
Kerberos login is independent from whether CDAP is running in Hadoop or not